packages/edit-site/src/components/editor/index.js

```diff
@@ -167,6 +170,9 @@ export default function EditSiteEditor( {
 		'edit-site-editor__loading-progress'
 	);
 
+	const [ paddingAppenderRef, paddingStyle ] = usePaddingAppender(
```

Comment:
Instead of duplicating the logic between post and site editor, Should we just move it to the "editor" component (there's already a number of similar hooks there)